knowledge-base/actions/chrnorm/deployment-status/action-security.yml

```diff
@@ -0,0 +1,9 @@
+name: "Deployment Status"
+github-token:
+  action-input:
+    input: token
+    is-default: false
+  permissions:
```

Comment:
The permission here should also be `deployment: write` and the reason would be `to update status of deployments`, view [here](https://github.com/chrnorm/deployment-status#usage-example)